### ISSUE TITLE
import is not deleted after import is done

### ISSUE THREAD

**Issue by qknight** [kartevonmorgen/FairSync#38]
Once the import is merged in clearance view normally it is removed from the database.

![grafik](https://github.com/user-attachments/assets/d28f7f49-fd83-494a-b5e7-e67a08285423)

This works if (1.) was selected as destination but not if (2.) was selected.

**Comment by qknight** [comment]
New plan: 

1. Once the merge was successfully done we delete all merging_items IDs from new_data_store
2. we delete selected_import ID from import_store

ClearanceView arguments:

```rust
    selected_import: RwSignal<Option<Id>>,
    import_store: RwSignal<HashSet<Id>>,
    merging_items: Signal<Vec<(Id, FairSyncRecord)>>,
    new_data_store: RwSignal<HashMap<Id, FairSyncRecord>>,
```

**Comment by qknight** [comment]
implemented in commit e7ce5f9588b85ebb1ab7206373ebbb32517865aa

    CleranceView: when the import was a success now the selected_import is removed also
